CMakeLists.txt

```
@@ -55,6 +55,18 @@ include(cmake/O3DEJson.cmake)
 include(cmake/Subdirectories.cmake)
 include(cmake/TestImpactFramework/LYTestImpactFramework.cmake) # Put at end as nothing else depends on it
 
+get_property(o3de_script_only GLOBAL PROPERTY "o3de_script_only")
+if (o3de_script_only)
+    if (NOT INSTALLED_ENGINE)
```

These two conditions could be checked in the same if statement.
```suggestion
if (o3de_script_only AND NOT INSTALLED_ENGINE)
```